pkg/targetgroupbinding/networking_manager_test.go

```diff
@@ -995,6 +995,214 @@ func Test_defaultNetworkingManager_computeRestrictedIngressPermissionsPerSG(t *t
 				},
 			},
 		},
+		{
+			name: "test for CIDRs",
+			fields: fields{
+				ingressPermissionsPerSGByTGB: map[types.NamespacedName]map[string][]networking.IPPermissionInfo{
+					types.NamespacedName{Namespace: "ns-1", Name: "tgb-1"}: {
+						"sg-a": {
+							{
+								Permission: ec2sdk.IpPermission{
+									IpProtocol: awssdk.String("tcp"),
+									FromPort:   awssdk.Int64(80),
```

Comment:
for CIDR based rules, the exact port values get used, could you modify the UTs as well so we can verify that the signle ports don't get merged into ranges.

Reply:
Modified. I also manually tested the CIDR case on EKS cluster, the single ports used did not get merged into ranges.